vendor/bat-native-ledger/src/transactions_info.cc

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/transactions_info.h"
+
+#include "base/json/json_reader.h"
+#include "base/json/json_writer.h"
+#include "base/values.h"
+
+namespace ledger {
+
+TransactionsInfo::TransactionsInfo() :
+    transactions({}) {}
+
+TransactionsInfo::TransactionsInfo(const TransactionsInfo& info) :
+    transactions(info.transactions) {}
+
+TransactionsInfo::~TransactionsInfo() = default;
+
+const std::string TransactionsInfo::ToJson() const {
+  base::Value dictionary(base::Value::Type::DICTIONARY);
```

Comment:
should be using the same approach that we currently use in ledger.cc for other things.

```
const std::string TransactionsInfo::ToJson() const {
  std::string json;
  braveledger_bat_helper::saveToJsonString(*this, json);
  return json;
}
```

Reply:
Confirmations has been refactored to use base::JSONReader/Writer and not RapidJSON, which was discussed a while ago, this code has been taken from Confirmations and I would recommend we keep this as all other code should eventually be refactored to use base where possible

Reply:
my concern is that now we have two approaches and we don't have defined approach which one to use. So this may confuse stuff.

Reply:
I think we should replace RapidJSON in one swipe